README.md

```diff
@@ -477,7 +477,7 @@ List of all available rules. The rules ported from `golint` are left unchanged a
 | [`exported`](./RULES_DESCRIPTIONS.md#exported)            |  []string   | Naming and commenting conventions on exported symbols.           |   yes    |  no   |
 | [`if-return`](./RULES_DESCRIPTIONS.md#if-return)           |  n/a   | Redundant if when returning an error.                            |   no    |  no   |
 | [`increment-decrement`](./RULES_DESCRIPTIONS.md#increment-decrement) |  n/a   | Use `i++` and `i--` instead of `i += 1` and `i -= 1`.            |   yes    |  no   |
-| [`var-naming`](./RULES_DESCRIPTIONS.md#var-naming)          |  whitelist & blacklist of initialisms   | Naming rules.                                                    |   yes    |  no   |
+| [`var-naming`](./RULES_DESCRIPTIONS.md#var-naming)          |  allow-list & blocklist of initialisms   | Naming rules.                                                    |   yes    |  no   |
```

Comment:
Shouldn't it be `allowlist` to be consistent with `blocklist`? I googled a little bit, it seems to be pretty common to use it as a single word.